src/Symfony/Bundle/FrameworkBundle/Command/CacheWarmupCommand.php

```diff
@@ -66,9 +66,9 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             $this->cacheWarmer->enableOptionalWarmers();
         }
 
-        $preload = $this->cacheWarmer->warmUp($cacheDir = $kernel->getContainer()->getParameter('kernel.cache_dir'));
+        $preload = $this->cacheWarmer->warmUp($kernel->getContainer()->getParameter('kernel.cache_dir'));
 
-        if ($preload && file_exists($preloadFile = $cacheDir.'/'.$kernel->getContainer()->getParameter('kernel.container_class').'.preload.php')) {
+        if ($preload && file_exists($preloadFile = $kernel->getContainer()->getParameter('kernel.build_dir').'/'.$kernel->getContainer()->getParameter('kernel.container_class').'.preload.php')) {
```

Comment:
I guess this is broken: build_dir might be read-only at this stage.
Instead, I suppose we need to skip this part unless build_dir === $cacheDir

Reply:
```suggestion
        if ($kernel->getContainer()->getParameter('kernel.build_dir') !== $cacheDir) {
            return;
        }
        if ($preload && file_exists($preloadFile = $cacheDir.'/'.$kernel->getContainer()->getParameter('kernel.container_class').'.preload.php')) {
```

Reply:
I did a change for that, in the 2nd commit, and somehow managed to get it reverted in the third 🤔 Fixing that